action.yml

```
@@ -0,0 +1,36 @@
+name: 'OpenTofu - Setup Tofu'
+description: 'Sets up OpenTofu CLI in your GitHub Actions workflow.'
+author: 'OpenTofu'
+inputs:
+  cli_config_credentials_hostname:
+    description: 'The hostname of a Terraform Cloud/Enterprise instance to place within the credentials block of the OpenTofu CLI configuration file. Defaults to `app.terraform.io`.'
+    default: 'app.terraform.io'
+    required: false
+  cli_config_credentials_token:
+    description: 'The API token for a Terraform Cloud/Enterprise instance to place within the credentials block of the OpenTofu CLI configuration file.'
+    required: false
+  tofu_version:
+    description: 'The version of OpenTofu CLI to install. If no version is given, it will default to `latest`.
+    default: 'latest'
+    required: false
+  tofu_wrapper:
+    description: 'Whether or not to install a wrapper to wrap subsequent calls of the `tofu` binary and expose its STDOUT, STDERR, and exit code as outputs named `stdout`, `stderr`, and `exitcode` respectively. Defaults to `true`.'
+    default: 'true'
+    required: false
+outputs:
+  stdout:
+    description: 'The STDOUT stream of the call to the `tofu` binary.'
+    value: ''
+  stderr:
+    description: 'The STDERR stream of the call to the `tofu` binary.'
+    value: 'Wrapper was not activated'
+  exitcode:
+    description: 'The exit code of the call to the `tofu` binary.'
+    value: '126'
```

```suggestion
  stdout:
    description: 'The STDOUT stream of the call to the `tofu` binary.'
  stderr:
    description: 'The STDERR stream of the call to the `tofu` binary.'
  exitcode:
    description: 'The exit code of the call to the `tofu` binary.'
```

Property `value` is not part of the [metadata syntax for GitHub Actions](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-docker-container-and-javascript-actions).

---

Thanks for spotting. It's a relict of the action definition I sketched to build tofu from source.

WDYT about leaving the values to have the defaults for when users will set `tofu_wrapper` to false?

---

I don't think `value` property itself is a part of the [GitHub's Action's metadata syntax](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-docker-container-and-javascript-actions) (outside of composite actions, which `action.yml` file is not).

As a result, the default values won't be returned in any configuration.

---

Thanks for the input. I'll apply suggested changes.